src/components/editor/Editor.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from "react";
+import PlateEditor from "./PlateEditor";
+import axios from "axios";
+import queryString from "query-string";
+
+import fm from "front-matter";
+
+//TODO: make this a configurable option in gatsby-config.js
+const CONTENT_BRANCH = "develop";
+
+export const Editor = () => {
+  const [value, setValue] = useState(null);
+  const [debug, setDebug] = useState(false);
+  useEffect(() => {
+    const get_file_from_github = async () => {
+      const parsed = queryString.parse(location.search);
+      if (parsed.file) {
+        const res = await client.get(
+          `${CONTENT_BRANCH}/content/${parsed.file}`
+        );
```

Comment:
If we are using await we should probably try catch this block and maybe show and error saying the page couldn't be fetched. 

i.g. if in the catch block we just return `<div>Something went wrong! Could not edit the page!</div>`